acf/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Yes they need to set up apache correctly as well since this is problem of how the server is setup where the serve should not list files even if is apache or Nginx.